src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  let preparedSelectedId = selectedTabId;
+
+  if (!tabs.find(tab => tab.id === selectedTabId)) {
+    preparedSelectedId = tabs[0].id;
+  }
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <>
+              <li
+                key={tab.id}
+                data-cy="Tab"
+                className={cn({ 'is-active': preparedSelectedId === tab.id })}
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => (preparedSelectedId === tab.id
+                    ? null
+                    : onTabSelected(tab))}
+                >
+                  {tab.title}
+                </a>
+              </li>
+              {preparedSelectedId === tab.id && (
+                <div className="block" data-cy="TabContent">
+                  {tab.content}
+                </div>
+              )}
```

Comment:
content should be outside of the list and outside of the .tabs element
actual:
![actual-tabs](https://github.com/mate-academy/react_tabs-js/assets/33029864/f9439e88-56f8-49da-9fad-882b60a0c60b)

working example:
![workink-tabs](https://github.com/mate-academy/react_tabs-js/assets/33029864/2fccca7d-3924-4885-abcf-f89ec5468e75)